share/spack/gitlab/cloud_pipelines/stacks/ml-darwin-aarch64-mps/spack.yaml

```diff
@@ -0,0 +1,139 @@
+spack:
+  view: false
+
+  concretizer:
+    unify: false
+    reuse: false
+
+  config:
+    concretizer: clingo
+    db_lock_timeout: 120
+    install_tree:
+      root: $spack/opt/spack
+      padded_length: 256
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  packages:
+    all:
+      require: target=aarch64
+      variants: +mps~cuda~rocm
+    mpi:
+      require: openmpi
+
+  specs:
+  # Hugging Face
+  - py-transformers
+
+  # JAX
+  - py-jax
+  # - py-jaxlib     # bazel codesign
+
+  # Keras
+  - py-keras-applications
+  - py-keras-preprocessing
+  - py-keras2onnx
+  # - py-keras      # bazel codesign
+
+  # MXNet
+  - mxnet
+
+  # PyTorch
+  - py-botorch
+  - py-gpytorch
+  - py-pytorch-gradual-warmup-lr
+  - py-segmentation-models-pytorch
+  - py-timm
+  - py-torch
+  - py-torch-cluster
+  - py-torch-geometric
+  - py-torch-sparse
+  - py-torchdata
+  - py-torchfile
+  - py-torchgeo
+  - py-torchvision
+
+  # scikit-learn
+  - py-scikit-learn
+  - py-scikit-learn-extra
+
+  # TensorBoard
+  - py-tensorboard
+  - py-tensorboard-data-server
+  - py-tensorboard-plugin-wit
+  - py-tensorboardx
+
+  # TensorFlow
+  # - py-tensorflow               # bazel codesign
+  # - py-tensorflow-datasets      # bazel codesign
+  # - py-tensorflow-hub           # bazel codesign
+  # - py-tensorflow-metadata      # bazel codesign
+  # - py-tensorflow-estimator     # bazel codesign
+  # - py-tensorflow-probability   # py-dm-tree due to bazel codesign
+
+  # XGBoost
+  - py-xgboost
+  - xgboost
+
+  # ERRORS
```

Comment:
What was the issue here? It builds fine for me on MPS. If you point me to an error log I can investigate. This can obviously happen after the PR is merged.

Reply:
The issue is for whatever reason a number of these specs concretize to depend on the older py-torch, and that py-torch does not build. Should look into those concretization+build errors once this is merged and update the stack as things are fixed.